### PR TITLE
add lilypond to installation

### PR DESCRIPTION
**Pull request by Juphex**
might be good to have it here

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ rosdep install --from-paths src --ignore-src -r -y
 
 # Install python dependencies
 pip3 install -r requirements.txt
+
+# Install lilypond dependencies that are necessary for visualization of the detected notes 
+sudo apt-get install lilypond
```
[the remark this answers is not in the document]
Comment (Flova) [reply README.md:49]: It seems like lilypond is not in the https://github.com/ros/rosdistro package index. But we can create a pull request there and add it to the index.

[the remark this answers is not in the document]
Comment (Juphex) [reply README.md:49]: I dont know if I did it correctly. https://github.com/ros/rosdistro/pull/36561

[the remark this answers is not in the document]
Comment (Flova) [reply README.md:49]: I commented on the other PR

[the remark this answers is not in the document]
Comment (Juphex) [reply README.md:49]: Alright lets see. Do we want to let this PR open?

[the remark this answers is not in the document]
Comment (Flova) [reply README.md:49]: We can merge this one and update the readme and dependency one the other PR is merged.

**Review by Flova — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Flova) [inline README.md:49]: Consider changing `apt-get` to `apt`

Comment (Juphex) [reply README.md:49]: That is actually the same as in their docs.

Comment (Flova) [reply README.md:49]: You can use both. `apt` is newer, a bit shorter and prettier (progressbar etc.). It is therefore preferred by most end-users nowadays. They use the same backend, so standard package installation works the same with both of them.

Comment (Flova) [reply README.md:49]: See https://itsfoss.com/apt-vs-apt-get-difference/

Comment (Flova) [reply README.md:49]: This is more fyi, you don't need to change it.

**Pull request merged**